src/js/modules/articles/components/ArticleFeaturedMedia.js

```diff
@@ -48,7 +56,7 @@ class ArticleFeaturedMedia extends Component {
       imgWidth: 0,
     };
   }
-  componentDidMount() {
+  componentDidMount = () => {
```

Comment:
Careful. Does this actually work? React lifecycle functions aren't like normal class functions

Reply:
Yes, I tested it. Basically, if the featured media is more vertical than horizontal, it'll be floated to the left of the body text and not span the whole width. You can see that behavior in the article page.